modules/minimap2/1.0/config/default.yaml

```diff
@@ -2,16 +2,23 @@ lcr-modules:
     
     minimap2:
 
-        # TODO: Update the list of available wildcards, if applicable
         inputs:
             # Available wildcards: {seq_type} {genome_build} {sample_id}
-            sample_fastq: "__UPDATE__"
-            reference_build: "__UPDATE__"
+            sample_fastq:
```

Comment:
I think this needs some documentation about the `{number}` wildcard and how to correctly specify paired and unpaired fastq files.